sdk/appconfiguration/azure-data-appconfiguration-v2/src/main/java/com/azure/v2/data/appconfiguration/models/Key.java

```diff
@@ -16,7 +16,7 @@
  * Keys serve as identifiers for key-values and are used to store and retrieve corresponding values.
  */
 @Metadata(properties = { MetadataProperties.IMMUTABLE })
-class Key implements JsonSerializable<Key> {
+final class Key implements JsonSerializable<Key> {
```

Comment:
This is a generated model. If we are adding `final`, it should be done through customization.  Do we need to make this `final` since this is not a public type?